Makefile

```diff
@@ -0,0 +1,21 @@
+LDFLAGS = -ldflags '-s -w'
+
+.PHONY: build
+build: host-switch
+
+.PHONY: host-switch
+host-switch:
```

Comment:
Just call this target `bin/host-switch.exe`, and you can drop the extra `.PHONY` and use `$@` in the command.